packages/flutter_tools/test/commands.shard/permeable/packages_test.dart

```diff
@@ -334,9 +332,9 @@ void main() {
       androidManifest.writeAsStringSync(updatedAndroidManifestString);
 
       final PackagesCommand command = await runCommandIn(projectPath, 'get');
-      final PackagesGetCommand getCommand = command.subcommands['get'] as PackagesGetCommand;
+      final PackagesGetCommand? getCommand = command.subcommands['get'] as PackagesGetCommand?;
```

Comment:
Might as well cast now
```suggestion
      final PackagesGetCommand getCommand = command.subcommands['get']! as PackagesGetCommand;
```